proto/cacheclient.proto

```diff
@@ -139,6 +139,16 @@ message _DictionarySetRequest {
 
 message _DictionarySetResponse {}
 
+message _IncrementRequest {
+  bytes cache_key = 1;
+  int64 increment_by = 2;
```

Comment:
```suggestion
message _IncrementRequest {
  bytes cache_key = 1;
  // Amount to add to the stored value.
  // If this key doesn't currently exist, it's created with this value (encoded as a base 10 string)
  int64 increment_by = 2;
```

I really like proto comments. They even make it into generated code in all languages I've used.

Reply:
Nice! Definitely agreed - haven't seen them before in our protos but they seem like a really nice thing to have. Thanks for these!